app/main.py

```
@@ -125,15 +142,23 @@ async def snapshot_builder(base_url: str) -> dict[str, object]:
         try:
             yield
         except asyncio.CancelledError:
             # Uvicorn may cancel lifespan tasks during reload/shutdown.
             # Treat this as a normal shutdown path.
             pass
         finally:
+            if sendspin_service:
+                try:
+                    await sendspin_service.stop()
+                except Exception:
+                    pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)